src/main/kotlin/com/epam/carapi/entity/Dealer.kt

```diff
@@ -0,0 +1,14 @@
+package com.epam.carapi.entity
+
+import javax.persistence.*
+
+@Entity
+@Table(name = "dealers")
+data class Dealer(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    val id: Long? = null,
```

Comment:
As `id` field cannot be `null`, lets remove `?` statement from `id` for all entities